### DISCUSSION TITLE
Supabase admin security for only backend modifications

### DISCUSSION THREAD

**Discussion by tscans** [supabase/supabase#4044] [Questions] [answered]
When I used firebase in the past, I bypassed using any rules by just making admin access only on my node server. I haven't found this option so far in the supabase documentation. I'd like to avoid using and making complex rules for row level security by just allowing my server to be the only entity that can change the tables.

Anyone know of a way to do that?

**Comment by liaujianjie** [comment, marked as answer]
This can be done by using the service key to create the Supabase client on your server. From the [docs](https://supabase.com/docs/guides/auth/row-level-security#never-use-a-service-key-on-the-client):

> Supabase provides special "Service" keys, which can be used to bypass all RLS.

You can grab your service key from the dashboard under `Settings > API > Project API keys > service_role`.
